### DISCUSSION TITLE
AlsoNotifyCanExecuteFor and AllowConcurrentExecutions missing from Preview 3

### DISCUSSION THREAD

**Discussion by golemme** [CommunityToolkit/Maui#389] [Q&A] [answered]
I'm using CommunityToolkit.Mvvm.ComponentModel and CommunityToolkit.Mvvm.Input in a .NET MAUI application, but some of the code generator functionality pertaining to ICommand seems to be missing. While AlsoNotifyChangeFor() is still there, AlsoNotifyCanExecuteFor() and the AllowConcurrentExecutions parameter for ICommand are missing. Do I need to install anything other than CommunityToolkit.Mvvm to get those?

**Comment by VladislavAntonyuk** [comment]
It’s probably better to ask this question here: https://github.com/CommunityToolkit/WindowsCommunityToolkit

**Reply by golemme** [reply]
Actually, @bijington identified the correct repo: [CommunityToolkit/dotnet](https://github.com/CommunityToolkit/dotnet)

**Comment by bijington** [comment, marked as answer]
Actually you want this repo:

https://github.com/CommunityToolkit/dotnet

**Reply by golemme** [reply]
Yes, I see my mistake now. Although the MVVM code generators are being touted as something newly available in .NET MAUI, the toolkit has been added to the dotnet repository of CommunityToolkit.

**Reply by bijington** [reply]
Thanks for the confirmation 

Yes so we have a big family of CommunityToolkit and functionality is placed where it fits best. The dotnet one is agnostic to frameworks such as .NET MAUI or WPF, etc. 

We are probably touting it to use in .NET MAUI apps as is reduces so much boiler plate code required to be written.

**Comment by CliffAgius** [comment]
Closed as answered
